File: service/src/main/java/net/huray/backend/minuting/repository/SpaceRepository.java
package net.huray.backend.minuting.repository;

import net.huray.backend.minuting.entity.Space;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface SpaceRepository extends JpaRepository<Space, Long> {

    List<Space> findByIsPublicTrue();
}

File: service/src/main/kotlin/net/huray/backend/minuting/repository/SpaceRepository.kt
package net.huray.backend.minuting.repository

import net.huray.backend.minuting.entity.SpaceEntity
import org.springframework.data.jpa.repository.JpaRepository

interface SpaceRepository : JpaRepository<SpaceEntity, Long> {

    fun findByIsPublicTrue(): List<SpaceEntity>

}